packages/SwingSet/misc-tools/all-delivery-time-computrons.py

```diff
@@ -0,0 +1,34 @@
+#
+# Given a slogfile on stdin and a vatID, this emits a CSV of every
+# delivery (including BOYD and GC actions), with their wallclock time
+# and computrons (if any). This can be turned into a scatter chart
+# which might show trends like organic GC taking longer over time. It
+# ignores replays.
```

Comment:
```suggestion
# Given a slogfile on stdin and a vatID argument, emit a CSV of every
# non-replay delivery to that vat:
#   crankNum,deliveryNum,durationInSeconds,computrons
# (in which computrons can be empty). This can be turned into a scatter
# chart which might show trends like organic GC taking longer over time.
```